Author: Michael Vogt <michael.vogt@ubuntu.com>
Description: Use metacity control center applet instead of compiz one.
 The rationale is that the ccp backend will transparently work with the
 metacity options under gnome.
Origin: vendor, ubuntu (1:0.5.1+git20070618-0ubuntu2)
--- compiz.orig/gtk/gnome/compiz.desktop.in
+++ compiz/gtk/gnome/compiz.desktop.in
@@ -5,7 +5,7 @@ _Name=Compiz
 Exec=compiz
 NoDisplay=true
 # name of loadable control center module
-X-GNOME-WMSettingsModule=compiz
+X-GNOME-WMSettingsModule=metacity
 # autostart phase
 X-GNOME-Autostart-Phase=WindowManager
 X-GNOME-Provides=windowmanager
